README.md

```diff
@@ -14,6 +14,9 @@ I use this project to learn about Linters, Github Workflows, Css grid/flexbox, i
 - CSS (grid, flexbox)
 - Visual Studio Code
 
+## Live Demo
+https://moise-mulungu.github.io/portfolio-mobile-version/
+
```

Comment:
- [ ] [OPTIONAL] - You can use this structure to display the link on your README file:
`[link](https://moise-mulungu.github.io/portfolio-mobile-version/)`
It will be displayed without the whole link - look pretty, isn`t it?
> This is just a sample:
<img width="266" alt="Screen Shot 2022-01-24 at 12 46 23" src="https://user-images.githubusercontent.com/49064106/150750217-1fc3c091-ac38-4a61-8285-6fecb917bea8.png">

Reply:
Yeah, that one looks really pretty. 
Thank you, the next time I will use it.

Reply:
@moise-mulungu thanks nice 👍

Reply:
@moise-mulungu After merging to main, the Demo live link is not working. Do not worry, you just consider it setting the root - fix it and set the main branch as a display root. If you have questions, I am open for discussing 💯